llama_stack/providers/adapters/safety/together/safety.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+import pydantic
+from together import Together
+
+import asyncio
+
+from llama_stack.distribution.request_headers import get_request_provider_data
+from .config import TogetherSafetyConfig
+from llama_stack.apis.safety import *
+import logging
+
+class TogetherHeaderInfo(BaseModel):
+    together_api_key: str
+
+
+class TogetherSafetyImpl(Safety):
+    def __init__(self, config: TogetherSafetyConfig) -> None:
+        self.config = config
+
+    async def initialize(self) -> None:
+        pass
+
+    async def run_shield(
+            self,
```

Comment:
formatting looks off -- could you install `pre-commit` and run it once?

Reply:
how to install pre-commit here?

Reply:
you can just do a `pip install pre-commit`. Then you can run `pre-commit install` which will install it as a git hook. Then when you make a commit, pre-commit will run its hook which includes linters, etc..

Reply:
done